.github/workflows/pr-open.yml

```diff
@@ -154,191 +142,46 @@ jobs:
           npm ci
           npm test
 
-  
-  trivy-repo:
-    name: Trivy-Repo
+  linting:
+    name: Linting-Code
     needs:
       - check
     if: needs.check.outputs.build == 'true'
     runs-on: ubuntu-18.04
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+      
+      - name: Tests
+        run: |
+          cd backend
```

Comment:
The backend is changing -- we're using Java now. It's not merged yet so you wouldn't have access to this. Let's comment out all the linting jobs for now and add them as new tasks after we get this merged.

Reply:
Commented.